apps/site/assets/js/test/trip-plan_test.js

```diff
@@ -136,6 +144,25 @@ describe("trip-plan", () => {
         `);
       });
 
+      it("gets element with getById", () => {
```

Comment:
To me as an outsider, I think this could include more about the query selector in the description, like maybe appending something along the lines of, "from within the container element".